src/test/kotlin/extensions/xu/rui/PrimitiveTypeNonNullableSupportTest.kt

```diff
@@ -0,0 +1,44 @@
+package extensions.xu.rui
+
+import com.winterbe.expekt.should
+import extensions.chen.biao.KeepAnnotationSupport
+import org.junit.Before
+import org.junit.Test
+import wu.seal.jsontokotlin.DefaultValueStrategy
+import wu.seal.jsontokotlin.PropertyTypeStrategy
+import wu.seal.jsontokotlin.generateKotlinDataClass
+import wu.seal.jsontokotlin.interceptor.InitWithDefaultValueInterceptor
+import wu.seal.jsontokotlin.interceptor.PropertyTypeNullableStrategyInterceptor
+import wu.seal.jsontokotlin.test.TestConfig
+
+class PrimitiveTypeNonNullableSupportTest {
+
+    private val json = """{ picture: { "id" : 1, "url" : "" } }"""
+
+    private val expectResult = """data class Test(
+    val picture: Picture? = null
+) {
+    data class Picture(
+        val id: Int = 0, // 1
+        val url: String? = null
+    )
+}"""
+
+
+    @Before
+    fun setUp() {
+        TestConfig.setToTestInitState()
+        TestConfig.propertyTypeStrategy = PropertyTypeStrategy.Nullable
+        TestConfig.defaultValueStrategy = DefaultValueStrategy.AllowNull
+    }
+
+    @Test
+    fun interceptTest() {
+        val kotlinDataClass = json.generateKotlinDataClass()
+        KeepAnnotationSupport.getTestHelper().setConfig("xu.rui.force_primitive_type_non-nullable", "true")
```

Comment:
Should it be `KeepAnnotationSupport`?

Reply:
It should change to PrimitiveTypeNonNullableSupport.

Reply:
> Should it be `KeepAnnotationSupport`?
It should change to PrimitiveTypeNonNullableSupport.